tools/integration-tests/tester/framework/config/config.go

```diff
@@ -31,76 +49,181 @@ type GoShimmer struct {
 	DRNG
 }
 
+// NewGoShimmer creates a GoShimmer config initialized with default values.
+func NewGoShimmer() (config GoShimmer) {
+	config = GoShimmer{}
+	fillStructFromDefaultTag(&config)
+	return
+}
+
+// fillStructFromDefaultTag recursively explores the given struct pointer and sets values of fields to the `default` as
+// specified in the struct's tags.
+func fillStructFromDefaultTag(s interface{}) {
```

Comment:
I would make this `fillStructFromDefaultTag(val reflect.Value)` and move the single `reflect` conversion inside the `NewGoShimmer`

Reply:
Should we maybe move that function into a different file to keep config.go concise as this probably requires updates more frequently.